02-props-state-component-architecture/state/randomcolors/src/boxes.js

```diff
@@ -0,0 +1,28 @@
+import React, { Component } from 'react';
+import Box from './box';
+
+class Boxes extends Component{
```

Comment:
having this `Boxes` component is a bit of an anti-pattern. You don't really need a component whose only purpose is to render x number of a different component.